### PR TITLE
minimal permission update

### PR DESCRIPTION
**Pull request by amalykhi**
[OCM-3521](https://issues.redhat.com/browse/OCM-3521)

### CODE REVIEW

**Comment by bardielle** [comment]
@arendej Please review it also

File: docs/index.md [hunk 1]
```diff
@@ -160,20 +160,7 @@ To use the Red Hat Cloud Services provider inside your Terraform configuration y
                 "Sid": "VisualEditor1",
                 "Effect": "Allow",
                 "Action": [
-                   "s3:PutObject",
-                   "s3:GetObject",
-                   "s3:AbortMultipartUpload",
-                   "iam:ListRoles",
```
Comment (nirarg) [inline docs/index.md:166]: all s3 actions are removed because added s3:*
Is it intended also removing iam and sts actions?

Comment (amalykhi) [reply docs/index.md:166]: updated the policy and forgot to add them to the index. Fixed

**Pull request merged**